esp/services/ws_config2/ws_config2_binding.hpp

```diff
@@ -27,4 +27,4 @@ class Cws_config2SoapBindingEx : public Cws_config2SoapBinding
 
 
 private:
-};
+};
```

Comment:
github's complaining about end of file. I think Richard created some git hooks you can apply to your git environment that will short circuit these types of issues before they get pushed to your remote

Reply:
I added a <cr> to the end. I do have the hooks added, but it did not find this. Perhaps they have been updated since I last added them. I will double check.